code/controllers/subsystem/job.dm

```diff
@@ -505,10 +505,36 @@ SUBSYSTEM_DEF(job)
 	job.give_donor_stuff(living_mob, M) // yogs - Donor Features
 	job.give_cape(living_mob, M)
 	job.give_map_flare(living_mob, M)
+	if(SSevents.holidays && SSevents.holidays["St. Patrick's Day"])
+		irish_override() // Assuming direct control.
+	else
+		job.give_bar_choice(living_mob, M)
```

Comment:
Should probably do the job check before calling this, instead of having the proc check.